docs/data/sdks/python/index.md

```diff
@@ -45,10 +61,7 @@ def callback_func(event, code, message=None):
   # code: status code of request response
   # message: a optional string message for more detailed information
 
-
 client.configuration.api_key = "new api key"
-client.configuration.flush_queue_size = 100
-client.configuration.flush_interval_millis = 10000 # 10 seconds
```

Comment:
Why remove flush queue size and interval here for Python? We have it in the other SDKs above